sklearn/linear_model/tests/test_logistic.py

```diff
@@ -90,6 +91,18 @@ def test_predict_3_classes():
     check_predictions(LogisticRegression(C=10), X_sp, Y2)
 
 
+def test_lr_liblinear_warning():
+    n_samples, n_features = iris.data.shape
+    target = iris.target_names[iris.target]
+
+    for LR in [LogisticRegression(solver='liblinear', n_jobs=2),
+               LogisticRegressionCV(solver='liblinear', n_jobs=2)]:
```

Comment:
Yep it seems I messed up there. I am the one to blame. Your original solution (not modifying `_check_solver_option`) was the one.

> I thought @glemaitre would be more familiar with the conventions used in scikit-learn, so I suggested him to make my requests more conformant with the conventions.

I am still not familiar with all the base code even if I should have catch that ;)